src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -149,8 +162,9 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                         if(c.Account.Primary_Contact__c == null) {
                             mapAccountIdContactId.put(c.AccountId, c.Id);
                         }
-                        if (c.Account.RecordTypeId == userDefinedHHRecordTypeId
-                                && UTIL_CustomSettingsFacade.getSettings().Automatic_Household_Naming__c != false) {
+                        if (c.Account.RecordTypeId != null
```

Comment:
I'd be curious to see if we do this check anywhere else besides just these two places and wonder if it's worth breaking that check out into a method. Its probably fine as is but if we keep adding to this class which is already pretty big, we may want to consider it in the future. (referring to this if check on lines 165-167)

Reply:
That's a great idea! I will create boolean methods for both household and admin.